### DISCUSSION TITLE
Termostato Decdeal

### DISCUSSION THREAD

**Discussion by Osky9** [indomus/forum#125] [HUB: Home Assistant]
Ciao a tutti,
ho appena installato un termostato Decdeal BHT6000, fino alla configurazione con l'app Tuya tutto ok, ma appena lo inserisco su HA la card mi indica la temperatura doppia. Anche sull'app Alexa la temperatura è corretta. 
Avete idea di cosa possa succedere?
Grazie mille

**Comment by Osky9** [comment]
Ok, da quanto posso constatare nessuna ha avuto il mio problema. Allora provo a chiedervi se posso risolverla in altra maniera. Posso impostare un valore fisso da togliere al valore nella visualizzato? Mi spiego meglio, se su lovelace visualizzo 40 gradi fare in modo che a questo valore venga sempre sottratto in determinato valore "x" così che io visualizzi la temperatura corretta. So che è una soluzione "alla buona" ma non capirei come trovarne un'altra in quanto sia sul display che sull'app "Smart life" la temperatura che visualizzo è corretta.
Grazie a tutti!

**Comment by Osky9** [comment]
Ok, ho risolto, bastava andare nelle opzioni dell'integrazione. Scusatemi.

**Reply by mriitan** [reply]
che intendi per opzione di integrazione? grazie
